images/pytorch-pods/launch_k8s_workers.py

```diff
@@ -75,6 +75,18 @@
 def _format_env(envs):
   return [{'name': k, 'value': v} for k, v in envs.items()]
 
+# Name must consist of lower case alphanumeric characters or '-'.
+def _sanitize_job_name(name):
```

Comment:
IMO it would be better to use `re.sub` or similar than to imperatively build a new string. ie something like `re.sub(r'[^a-z0-9\-]', '-',  name.lower())`

Reply:
SGTM

Reply:
tested again locally